extension/js/common/core/crypto/key.ts

```diff
@@ -221,9 +222,9 @@ export class KeyUtil {
           allErr.push(e as Error);
         }
       }
-      if (err) {
+      /* todo: re-throw? if (err) {
         allErr.push(...err);
-      }
+      } */
```

Comment:
I wanted to allow reading the results partially. Like if WKD sends a mixture of keys that some throw when processing and some don't, I'd like to see the valid keys and then the errors side by side (code-wise), so that the app may choose to still use the valid ones and ignoring the errors. That's why we are returning errors instead of throwing them, unlike everywhere else in the codebase (I suppose this is what your comment was about).

Reply:
The results may differ compared to 4.10.10 implementation so we need to decide how to deal with this (see comment above).

Reply:
let's file the comment above for another issue.